### PR TITLE
Avh

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

File: .github/workflows/cmsis_rv2.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    - name: Install Python AVH Client
+      run: |
+        pip install -e AVH/infrastructure/python_resources
+
+    - uses: ammaraskar/gcc-problem-matcher@master
+
+    - name: Run tests
+      run: |
+        arm-vhclient -b aws execute --specfile Project/avh.yml
```
Comment (JonatanAntoni) [inline .github/workflows/cmsis_rv2.yml:63]: Given the above env variables are set you can run the very same command on your local machine.

File: Project/avh.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - -:Project/RTOS2_Validation-*.zip
+  - -:Project/RTOS2_Validation-*.junit
+  - -:Project/**/Output/**/*
+  - -:Project/**/RTE/**/RTE_Components.h
+  - Source/**/*
+steps:
+  - run: |
+      cd Project
+      pip install -r requirements.txt
+      python build.py --verbose build run
```
Comment (JonatanAntoni) [inline Project/avh.yml:20]: Also this one can be run on the local machine if the environment is setup properly.

File: Project/build.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+
+@matrix_command()
+def unzip(archive):
+    return ["bash", "-c", f"unzip {archive}"]
+
+
+@matrix_command()
+def cbuildsh(project):
+    return ["bash", "-c", f"cbuild.sh --quiet {project}"]
```
Comment (JonatanAntoni) [inline Project/build.py:102]: CMSIS-Build needs to be on PATH

File: Project/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (JonatanAntoni) [inline Project/requirements.txt:1]: Python requirements need to be installed

**Review by VladimirUmek — APPROVED** [review]
Looks good. Some things will probably change when we introduce support for different compilers and devices but I hope the changes will be minor.

**Pull request merged**